workflows/csharp/sdk/README.md

```diff
@@ -0,0 +1,43 @@
+# Dapr workflows
+
+In this quickstart, you'll create a simple console application to demonstrate Dapr's workflow programming model and the workflow management API. The console app starts and manages the lifecycle of a workflow that stores and retrieves data in a Dapr state store.
```

Comment:
Can we describe what the workflow does here? What is the order process workflow logic and what do each of the activities represent? That way the reader understands what the code is doing and the expectations from running the workflow